applications/minotari_console_wallet/src/lib.rs

```diff
@@ -106,6 +106,7 @@ pub fn run_wallet(shutdown: &mut Shutdown, runtime: Runtime, config: &mut Applic
         profile_with_tokio_console: false,
         view_private_key: None,
         spend_key: None,
+        reachability: None,
```

Comment:
Maybe `Some(ReachabilityMode::Private)`  for the wallet?

Reply:
Agree, wallets will in general not have a public IP

Reply:
This is the cli override though, set the default in https://github.com/sdbondi/tari/blob/ff17e520306c5f5993014e455d46a3d44d6de829/base_layer/wallet/src/config.rs